test-harness/build.gradle

```diff
@@ -26,3 +26,23 @@ task sourcesJar(type: Jar) {
 artifacts {
   archives jar, javadocJar, sourcesJar
 }
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      pom.project {
+        name 'Test Harness for SpotBugs Plugin'
+        description 'A test harness library for SpotBugs plugin developers'
+
+        licenses {
+          // This project contains copied source files from SonarSource and find-sec-bugs,
+          // which is licensed under LGPL 3.0, so we should use v3.0 instead of 2.1.
```

Comment:
Thanks for reply!

> This segment could be removed : https://github.com/spotbugs/spotbugs/blob/1aa089627fb3639634d7cfa4dbfeaf063e9917cd/findbugs/src/junit/org/sonar/plugins/findbugs/resource/SmapParser.java#L2

Oh I didn't find that this java file has two comments about license... I will remove the 1st one (LGPL 3.0 for SonarQube Findbugs Plugin), and keep the 2nd one (Copyright (c) 2012 Michael Schierl).